docgen/mdRenderer.js

```diff
@@ -14,7 +14,7 @@ const md =
     permalink: true,
     permalinkClass: 'anchor',
     permalinkSymbol: '',
-    // generate proper getting-started.html#install hrefs since we are
+    // generate proper https://www.algolia.com/doc/guides/building-search-ui/getting-started/ios/#install hrefs since we are
```

Comment:
Not sure this hack is needed anymore now we don't have `getting-started.html` at all :upside_down_face: